crates/astria-core/src/execution/v1alpha2/mod.rs

```diff
@@ -19,12 +19,18 @@ impl GenesisInfoError {
     fn incorrect_rollup_id_length(inner: IncorrectRollupIdLength) -> Self {
         Self(GenesisInfoErrorKind::IncorrectRollupIdLength(inner))
     }
+
+    fn no_rollup_id() -> Self {
+        Self(GenesisInfoErrorKind::NoRollupId)
+    }
 }
 
 #[derive(Debug, thiserror::Error)]
 enum GenesisInfoErrorKind {
-    #[error("`rollup_id` field did not contain a valid rollup ID")]
+    #[error("`rollup_id` field contained an invalid rollup ID")]
     IncorrectRollupIdLength(IncorrectRollupIdLength),
+    #[error("no `rollup_id` provided")]
```

Comment:
This should call out that the field was not set (technically the same, but most other similar errors in the codebase do it):
```suggestion
    #[error("`rollup_id` was not set")]
```